common/db.go

```diff
@@ -223,6 +265,45 @@ func (db *DB) LastSeenBlock() (*big.Int, error) {
 	return big.NewInt(lastSeenBlock), nil
 }
 
+func (db *DB) UpdateOrchs(orch *DBOrch) error {
+	if db == nil {
+		return nil
+	}
+	_, err := db.updateOrchs.Exec(orch.ServiceURI, orch.EthereumAddr)
+	if err != nil {
+		glog.Error("db: Unable to update orchestrator ", err)
+	}
+
+	return err
+}
+
+func (db *DB) SelectOrchs() ([]*DBOrch, error) {
+	if db == nil {
+		return nil, nil
+	}
+
+	rows, err := db.selectOrchs.Query()
+	defer rows.Close()
+	if err != nil {
+		glog.Error("db: Unable to get orchestrators updated in the last 24 hours", err)
+		return nil, err
+	}
+	orchs := []*DBOrch{}
+	for rows.Next() {
+		var orch DBOrch
+		var serviceURI string
+		var ethereumAddr string
+		if err := rows.Scan(&serviceURI, &ethereumAddr); err != nil {
```

Comment:
We can avoid using the serviceURI and ethereumAddr variables here, eg with `rows.Scan(&orch.ServiceURI, &orch.EthereumAddr)`